kedro/runner/parallel_runner.py

```diff
@@ -270,7 +270,6 @@ def _run(  # pylint: disable=too-many-locals,useless-suppression
 
```

Comment:
Could we leave this out as a seprate PR addressing KED-2780? I know it will end up being two very small PRs, but this way our git history will tell a more comprehensive story.